modular_zubbers/code/modules/species_refits_awareness/code/refithelper.dm

```diff
@@ -0,0 +1,59 @@
+
+/**
+ * Builds and returns a list of strings in accordance with the species names that an outfit has resprites for.
+ *
+ * This builds up a full set of /datum/data/vending_products from the product list of the vending machine type
+ * Arguments:
+ * * icon_state - The iconstate of the item. Usually, call this by the second argument, item.icon_state
+ * * obj/item - The item path that we'll operate on, to check for refits.
+ */
+/proc/check_for_refits(icon_state, var/obj/item/I)
```

Comment:
Please change var/obj/item/I to a different arg, no single letter variables.

Otherwise, looks good!